coding-standards.php

```diff
@@ -0,0 +1,222 @@
+            <?php else: ?>
+                <!--4th, if no banner image is specified either by text or file upload(image gallery image), then use default image fallback -->
+                <img src="<?php echo do_shortcode('[ghcmThemeImg]'); ?>services/healthcare-mgmt/banner.jpg" width="1224" height="447" alt="" />        
+            <?php endif; ?>    
+        <?php endif; ?>                   
+        <span class="edge left"></span>
+        <span class="edge right"></span>
+    </div>
+</div>
+<div id="carousel-example-generic" class="carousel slide hidden-md hidden-lg BsBannerMarginTop" data-ride="carousel">
+    <!-- Wrapper for slides -->
+    <div class="carousel-inner" role="listbox">
+        <div class="item active">
+            <?php if (get_field("glossary_content_banner_image_url_path") and get_field("glossary_content_banner_image_url_upload")): ?>
+                <!-- 1st, If both values exist, use the uploaded file path-->
+                <img src="<?php the_field('glossary_content_banner_image_url_upload'); ?>" alt="" />    
+            <?php else: ?>      
+                <?php if (get_field("glossary_content_banner_image_url_upload")): ?>
+                    <!-- 2nd, both values don't exist, so if the upload value exists, use this-->
+                    <img src="<?php the_field('glossary_content_banner_image_url_upload'); ?>" alt="" />    
+                <?php elseif (get_field("glossary_content_banner_image_url_path")): ?>
+                    <!-- 3rd, both values don't exist, so if the text path value exists, use this-->
+                    <img src="<?php echo do_shortcode('[ghcmThemeImg]'); ?><?php the_field('glossary_content_banner_image_url_path'); ?>"  alt="" /> 
+                <?php else: ?>
+                    <!--4th, if no banner image is specified either by text or file upload(image gallery image), then use default image fallback -->
+                    <img src="<?php echo do_shortcode('[ghcmThemeImg]'); ?>services/healthcare-mgmt/banner.jpg" alt="" />        
+                <?php endif; ?>    
+            <?php endif; ?>    
+            <div class="carousel-caption">            
+            </div>
+        </div>
+    </div>
+    <span class="contentRespBannerLeftEdge"></span>
+    <span class="contentRespBannerRightEdge"></span>
+</div>
+<script type="text/javascript">
+    alignBannerImage();
+</script>
+
+<?php
+$page_header = get_field('glossary_page_header_title');
+
+if (have_rows('glossary_page_breadcrumb')) {
+    $bread_crumb_html = '<div class="row"><div class="col-xs-12 col-sm-12 col-md-12 col-lg-12"><div class="clear full breadcrumbs">';
+    $bread_crumb_html .= '<ul class="ghn breadcrumbs">';
+    // loop through the rows of data
+    $index_of_loop = 0;
+    $number_of_rows = count(get_field('glossary_page_breadcrumb'));
+    while (have_rows('glossary_page_breadcrumb')) : the_row();
```

Comment:
You should convert this `while` statement to use curly braces instead. Only use this format when using `while` on one line.

e.g. This is okay:
```
<?php while: ?>
    <!-- some html here -->
<?php endwhile; ?>
```

But this isn't:
<?php
while:
    // ...
endwhile;
?>